modules/indexing/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryDetailMetrics.java

```diff
@@ -64,6 +64,12 @@
     /** Sum of execution time of completions time. */
     private long lastStartTime;
 
+    /** Name of schema. */
+    private String schema;
+
+    /** Flag of local query. */
+    private Boolean local;
```

Comment:
Do we really need object? Why not use primitive boolean?

Reply:
New fields have been removed. Failed to collect new metrics without compatibility break.